Adds Models section (#82)
kluster-ai/docs: get-started/models.md

---
title: Models
description: Learn what models are supported by kluster.ai API, main characteristics and API request limits for each model for both free and standard tiers.
---

# Models on kluster.ai

[kluster.ai](https://kluster.ai){target=\_blank} offers a wide variety of open-source models for both real time and batch inferences, with more being constantly added. 

> 

Reviewer: ```suggestion
```

> [kluster.ai](https://kluster.ai){target=\_blank} offers a wide variety of open-source models for both real time and batch inferences, with more being constantly added. 
> 
> This page covers all the different models that are supported by the API, with the API request limits for each.

Reviewer 2: ```suggestion
[kluster.ai](https://kluster.ai){target=\_blank} offers a wide variety of open-source models for both real-time and batch inferences, with more being constantly added. 

This page covers all the models the API supports, with the API request limits for each.
```

## Model names

> Each model supported by kluster.ai has a unique model that must be used when defining the `model` in the request.

Reviewer 2: ```suggestion
Each model supported by kluster.ai has a unique name that must be used when defining the `model` in the request.
```

|        Model         |                 Model API name                 |
|:--------------------:|:----------------------------------------------:|

> |   **DeepSeek R1**    |           ` deepseek-ai/DeepSeek-R1`           |
> |   **DeepSeek V3**    |           ` deepseek-ai/DeepSeek-V3`           |
> | **DeepSeek V3 0324** |        ` deepseek-ai/DeepSeek-V3-0324`         |

Reviewer 2: ```suggestion
|   **DeepSeek R1**    |            `deepseek-ai/DeepSeek-R1`           |
|   **DeepSeek V3**    |            `deepseek-ai/DeepSeek-V3`           |
| **DeepSeek V3 0324** |         `deepseek-ai/DeepSeek-V3-0324`         |
```

|   **Gemma 3 27B**    |            `google/gemma-3-27b-it`             |
|   **Llama 3.1 8B**   |  `klusterai/Meta-Llama-3.1-8B-Instruct-Turbo`  |
|  **Llama 3.1 405B**  | `klusterai/Meta-Llama-3.1-405B-Instruct-Turbo` |
|  **Llama 3.3 70B**   | `klusterai/Meta-Llama-3.3-70B-Instruct-Turbo`  |

> |   **Qwen 2.5 7B**    |         `Qwen/Qwen2.5-VL-7B-Instruct `         |

Reviewer 2: ```suggestion
|   **Qwen 2.5 7B**    |         `Qwen/Qwen2.5-VL-7B-Instruct`          |
```
